### ISSUE TITLE
Not working when copying demo StatelessWidget code

### ISSUE THREAD

**Issue by WilliamD47** [BirjuVachhani/adaptive_theme#44]
Hi, I changed the code slightly so that it ignores bold text in iOS accessibility and now it doesn't work even if i take away the modifications. here is my code:

```dart
class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return AdaptiveTheme(
        light: ThemeData(
          brightness: Brightness.light,
          primaryColor: Colors.red,
        ),
        dark: ThemeData(
          brightness: Brightness.dark,
          primaryColor: Colors.red,
        ),
        initial: AdaptiveThemeMode.dark,
        builder: (theme, darkTheme) => MediaQuery(
            data: MediaQueryData.fromWindow(WidgetsBinding.instance.window)
                .copyWith(boldText: false),
            child: MaterialApp(
              useInheritedMediaQuery: true,
              title: 'Reggie',
              theme: theme,
              darkTheme: darkTheme,
              home: const MyHomePage(title: 'Reggie'),
            )));
  }
}
```
![Screenshot 2022-07-29 at 21 18 47](https://user-images.githubusercontent.com/69767121/181837241-b705e915-6062-493b-b5e4-c276d0261dfb.png)

Here are simulator screenshots. Running in debug mode using VSCode to start

The accent colour which should be on the top appbar does not appear as red and neither does the floatingActionButton.

**Comment by BirjuVachhani** [comment]
@WilliamD47 I checked the code you have provided. It's not a problem with the package. You're setting your theme wrong, that's why it's not meeting your expectations. Replace `primaryColor` with `primarySwatch` and it should work the way you're expecting it to work.
